src/agoradatatools/etl/utils.py

```diff
@@ -53,7 +53,10 @@ def _get_config(config_path: str = None) -> list:
         raise yaml.scanner.ScannerError(
             "YAML file unable to be scanned.  Please provide a valid YAML file."
         )
-
+    if not isinstance(config, dict):
+        raise ValueError(
+            "YAML file must be loaded as a single dictionary.  Please reformat your YAML file correctly."
+        )
```

Comment:
Can you please add a unit test for this case?

Reply:
Yes. I added a unit test.